techeria_controller/techeria_app/models.py

```diff
@@ -90,6 +90,18 @@ class Meta:
         db_table = 'camera'
 
 
+class Accessories(models.Model):
+    image = models.ImageField(null=False, blank=False, upload_to='images/')
+    name = models.CharField(max_length=255)
+    price = models.FloatField()
+    description = models.TextField()
```

Comment:
Its better not to use null = False, because seller may forgot to upload the picture